c/clang/_demo/castdump/astdump.go

```diff
@@ -3,6 +3,7 @@ package main
 import (
```

Comment:
move to `chore/castdump`.

Reply:
> move to `chore/castdump`.

In progress, also do we need to dump it to json via cjson